### ISSUE TITLE
Click icon to copy address did not respect to address format setting

### ISSUE THREAD

**Issue by xlc** [polkadot-js/extension#184]
Configured to use Kusama address format, click the address icon and copied Polkadot address

`5CMio84PTX3LbcWyDcZYvuhBBMBDZw3JQaZV8b2RbQQ6a3Wi`

It should be the Kusama address starts with CsL

![Screen Shot 2019-10-25 at 7 29 57 PM](https://user-images.githubusercontent.com/1201310/67548308-038bf080-f75e-11e9-81a6-fa12d58f20c8.png)

**Comment by ltfschoen** [comment]
I just followed some instructions to generate a "Polkadot (live)" address.
I installed the [Polkadot JS Extension for Firefox](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/).
In the "ACCOUNTS" page I clicked "Options", which took me to the "SETTINGS" page, where I changed it to "Polkadot (live)" from the selection box of the "display addresses formatted for".
I then went back to "ACCOUNTS" and clicked the button "I want to create a new account with a new seed", which generated the 12-word seed, and I entered a description, and a password to encrypt it.
Now on the accounts page it shows the address "15HKpZLoZiUjrSWiVwgU2m7EigUctz4aaPWiD...", which is clipped at the end, so I don't know what it is or how to view the whole address.
And if I click the account identicon to copy the address to my clipboard and share it, it copies an address that is prefixed with "5" instead of "1" Polkadot (live) (e.g. 5HPAgD5jhwDGQuGHYJdTtcP4s4UyCgWSVtnU2tcoNU8B3bbP)
I've also tried exporting it to a file, but the file only shows the address that starts with "5".
